ihmc-perception/src/main/java/us/ihmc/sensors/zed/ZEDImageSensor.java

```diff
@@ -72,6 +72,7 @@ public class ZEDImageSensor extends ImageSensor
    private long grabSequenceNumber = 0L;
    private Instant lastGrabTime;
    private boolean lastGrabFailed = false;
+   private long currentSensorTimestamp;
```

Comment:
Seems more like a...
```suggestion
   private long lastGrabTimestamp;
```

Reply:
Fixed! Don't need a getCurrentSensorTimestamp method